BirthdayGreetingsTest/api.js

```diff
@@ -64,22 +59,22 @@ app.put("/birthdays/add-friend", (req, res) => {
   const { lastName, firstName, dateOfBirth, email } = req.body;
```

Comment:
This endoint could better meet conventions. Both the path and the type of request. When should you use a PUT vs POST?

Reply:
@RichardWatkins1 I guess we use POST here instead, still a little bit confused about this maybe we can discuss in the meeting?